app/views/public_issues/show_reviewers.html.haml

```diff
@@ -0,0 +1,12 @@
+.scroll-panel-header
+  = link_to "Wróć do numeru", public_issue_path, class: %w{btn btn-default btn-sm pull-right}
```

Comment:
To nie do końca może tak działać "w realu" bo ujawniałoby w praktyce dane recenzentów. Ale zostawmy taką wersję.